sdk/formrecognizer/Azure.AI.FormRecognizer/samples/README.md

```diff
@@ -0,0 +1,24 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-form-recognizer
+name: Azure Form Recognizer samples for .NET
+description: Samples for the Azure.AI.FormRecognizer client library
+---
+
+# Azure Form Recognizer client SDK Samples
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to recognize form fields, text, and tables in form documents. It includes the following capabilities:
+
+- Recognize form content - Recognize and extract tables, lines and words in forms documents, without the need to train a model.
+- Recognize receipts - Recognize and extract common fields from US receipts, using a pre-trained receipt model.
+- Recognize custom forms - Recognize and extract form fields and other content from your custom forms, using models you train with your own form types.
+
+You can find samples for each of this main functions below.
+- [Recognize form content](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md)
+- [Recognize receipts](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md)
+- [Recognize custom forms](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md)
+- [Train a model](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeFormContent.md)
```

Comment:
We consolidated this to one training sample for the README, but I believe there are two written ... one for training with forms and one for training with forms and labels.

Reply:
you are right. let me see how to include that. Thank you!